nix/proto-lens-pkg.nix

```diff
@@ -0,0 +1,23 @@
+{ mkDerivation, base, bytestring, containers, deepseq, ghc-prim
+, hpack, lens-family, parsec, pretty, primitive, profunctors
+, QuickCheck, stdenv, tagged, tasty, tasty-quickcheck, text
+, transformers, vector
+}:
+mkDerivation {
```

Comment:
The same with ghc-source-gen update. I guess maintainers need to that changes them selfs. Was it something not matching update? Or you needed fixes?

Reply:
I did revert of ghc-source-gen upgrade, this was redundant